index.js

```diff
@@ -170,13 +180,21 @@ app.get(`${baseRoute}export/jobs/:jobId`, getJobsHandler());
 app.get(`${baseRoute}export/jobs/:jobId/results/Output_File`, getJobsHandler('/results/Output_File'));
 
 const MOVED_PERMANENTLY = 301;
-app.get('/', (_, response) => response.redirect(MOVED_PERMANENTLY, 'https://github.com/agrc/serverless-print-proxy'));
+app.get('/', (_, response) => response.redirect(MOVED_PERMANENTLY, 'https://github.com/agrc/serverless-print-proxy#readme'));
 
 const port = process.env.PORT || DEFAULT_PORT;
 if (process.env.NODE_ENV !== 'test') {
-  app.listen(port, () => {
-    console.log(`printproxy listing on port ${port}`);
-  });
+  if (process.env.NODE_ENV === 'development') {
+    const credentials = {
+      key: fs.readFileSync('./localhost-key.pem'),
+      cert: fs.readFileSync('./localhost.pem'),
```

Comment:
**question**: this gives you a self signed cert?

Reply:
`mkcert` generates it. This wires it up to the express server.